### PR TITLE
modified whoami.c

### PR DESCRIPTION
**Pull request by Jerryh001**
1022997

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("1022997");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by Jerryh001** [comment]
Here comes an empty line

**Comment by jrjang2** [comment]
Merge your two commits to one before doing pull request.

Tip:
1. Try to use ```git rebase```.
2. Use ```git reset --mixed```, and commit again.

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Jerryh001** [comment]
all done(i guess)

**Comment by jrjang2** [comment]
Great. Try to merge three commits to one before doing pull request.

Tips:
1. ```git rebase```
2. ```git reset --mixed``

**Comment by Jerryh001** [comment]
Like this?
btw, when I do rebase
"Merge pull request #13 from yzliao123/master " commit has been eaten,is this OK?

**Comment by jrjang2** [comment]
Good job!

**Pull request merged**